src/pages/User/components/PullRequests/index.js

```diff
@@ -35,24 +35,36 @@ export default function PullRequests({ username }) {
   if (error || data.error_description) {
     return <ErrorText errorMessage={getErrorMessage()} />;
   }
-
-  const isComplete = data.prs.length >= pullRequestAmount;
+  let isComplete;
+  if (data.prs) {
+    isComplete = data.prs.length >= pullRequestAmount;
+  } else if (data.mrs) {
+    isComplete = data.mrs.length >= pullRequestAmount;
+  }
```

Comment:
See note above, but I'm thinking we may not need all the ifs/ternaries in this file.

Reply:
Will think about this. if you have any suggestion let me know.

Reply:
I think if we always assume prs will contains prs _and_ mrs we can just always use `prs` here.

Reply:
the issue here is in api/controllers/pr/index.js

      if (isGitHubUser) {
        const data = {
          prs,
          username,
          userImage: user.data.avatar_url,
        };

        res.json(data);
      } else if (isGitLabUser) {
        const data = {
          mrs,
          username,
          userImage: user[0].avatar_url,
        };

        res.json(data);
      }

This piece of code changes the attributes of the data object.

Reply:
What about something like?
```js
  const data = {
    prs,
    username,
    userImage: isGitHubUser ? user.data.avatar_url : user[0].avatar_url,
  };
```